scripts/run_performance_job.py

```diff
@@ -738,6 +738,15 @@ def run_performance_job(args: RunPerformanceJobArgs):
             raise Exception("Built apps directory must be present for Android benchmarks")
         getLogger().info("Copying Android apps to payload directory")
         shutil.copy(os.path.join(args.built_app_dir, "androidHelloWorld", "HelloAndroid.apk"), os.path.join(root_payload_dir, "HelloAndroid.apk"))
+
+        android_binlog_dir = os.path.join(root_payload_dir, "androidHelloWorldBinlog")
+        shutil.copytree(os.path.join(args.built_app_dir, "androidHelloWorldBinlog"), android_binlog_dir)
+
+        for file in glob(os.path.join(android_binlog_dir, "**", "*.binlog"), recursive=True):
```

Comment:
Why does this break on the first file rather than moving all the files or checking in an if for only one file and moving that one?

Reply:
That was copied from the existing block. I’ve updated it to copy only a single file.